### ISSUE TITLE
cannot run the go-instabot

### ISSUE THREAD

**Issue by rikinhim** [tducasse/go-instabot#27]
inside the go-instabot directory , i try to run go-instabot -run but returned "command not found".

then I discovered a file named "go-instabot", double click it and popup another terminal to start running for a sec, then return tell me that no such file or directory of "config/config.json", which I did have it.

I am running on Mac OS , kindly advise, thanks.

**Comment by rikinhim** [comment]
hello, the config folder with config.json file is in the same directory of
the script, and actually I can't see a bin folder, please kindly help.
Thanks

Lazaros Anastasiou <notifications@github.com> 於 2018年10月21日 週日 上午12:26寫道：

**Comment by anastalaz** [comment]
If you run the command 'go get github.com/tducasse/go-instabot' a binary file should be created in the /bin directory (look [here](https://golang.org/doc/code.html) on how the go directory hierarchy should look like).

In which directory did you find the go-instabot file?

**Comment by tducasse** [comment]
Try to follow the solution exposed in issue #14

**Comment by rikinhim** [comment]
thanks but I am using MAC and seem there is no need to set PATH ?

Thibaud Ducasse <notifications@github.com> 於 2018年10月23日 週二 上午6:57寫道：

**Comment by rikinhim** [comment]
Thanks I followed your link and found the bin folder and there is
a go-instabot file there! I actually had another go-instabot file in go/src/
github.com/tducasse/go-instabot/ , this is the place where I run this file
(by double-clicking it).

I tried to move the config folder with config.json to /bin and run(double
click it) the go-instabot , still no luck . Here is the message prompt:

==================================

users-MBP-2:~ user$ /Users/user/go/bin/go-instabot ; exit;

2018/10/23 13:37:46 Error reading config file, open config/config.json: no
such file or directory

logout

Saving session...

...copying shared history...

...saving history...truncating history files...

...completed.


==================================



Lazaros Anastasiou <notifications@github.com> 於 2018年10月23日 週二 上午2:55寫道：

**Comment by tducasse** [comment]
When you launch the command, you need to have a folder called `config`, and a file `config.json` inside this folder.

**Comment by rikinhim** [comment]
yes I did, and the file "go-instabot" is located together with the config
folder, and the config.json is inside this folder, I don't know why it says
"no such file or directory"

Thibaud Ducasse <notifications@github.com> 於 2018年10月23日 週二 下午1:50寫道：

**Comment by tducasse** [comment]
It doesn't need to be where `go-instabot` is, but it needs to be where you call the command.
Let's say go-instabot is in `/folder1/folder2/go-instabot`, and you launch the command from `/folder3`, then the `config` folder needs to be in `folder3/`.

**Comment by rikinhim** [comment]
I am getting your meaning now, and seems something is working , let me see..

Thibaud Ducasse <notifications@github.com> 於 2018年10月23日 週二 下午1:55寫道：

**Comment by rikinhim** [comment]
I have set the PATH like below , with the last one just added

------------------------------------------------------------

users-MBP-2:~ user$ echo $PATH

/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/go/bin:/Users/user/go/bin
------------------------------------------------------------

now ran the go-instabot -run, there a new response  instead of "no such
file or directory" >

------------------------------------------------------------

users-MBP-2:go-instabot user$ go-instabot -run

2018/10/23 14:16:35 Using config: config/config.json


2018/10/23 14:16:36 ERROR:The account is logged out
------------------------------------------------------------

kindly help again, thanks!!

Wai Li <rikinhim@gmail.com> 於 2018年10月23日 週二 下午2:14寫道：
